src/StatusQ/Core/StatusScrollView.qml

```diff
@@ -0,0 +1,20 @@
+import QtQuick 2.14
+import QtQuick.Templates 2.14 as T
+
+import StatusQ.Controls 0.1
+
+Flickable {
```

Comment:
Please use ScrollView as the base class, https://doc.qt.io/qt-5/qml-qtquick-controls2-scrollview.html

and customize it accordingly: https://doc.qt.io/qt-5/qtquickcontrols2-customize.html#customizing-scrollview